pkg/querier/queryrange/codec.go

```diff
@@ -1005,6 +1005,10 @@ func encodeResponseJSONTo(version loghttp.Version, res queryrangebase.Response,
 		if err := marshal.WriteVolumeResponseJSON(response.Response, w); err != nil {
 			return err
 		}
+	case *logproto.VolumeResponse:
+		if err := marshal.WriteVolumeResponseJSON(response, w); err != nil {
+			return err
+		}
```

Comment:
Really? We use `logproto.VolumeResponse` and `VolumeResponse`?

Reply:
I've just checked. We seem to handle both :see_no_evil: We probably should only use `logproto.VolumeResponse` and throw the other out.